src/core_plugins/kibana/public/discover/partials/no_results.html

```diff
@@ -28,19 +36,35 @@
     </div>
 
     <div ng-show="opts.timefield">
+      <h3>
+        Expand your time range
+      </h3>
+
       <p>
-        <h3>Expand your time range</h3>
-        <p>I see you are looking at an index with a date field. It is possible your query does not match anything in the current time range, or that there is no data at all in the currently selected time range. Click the button below to open the time picker. For future reference you can open the time picker by clicking on the <a class="kuiButton kuiButton--primary kuiButton--small" ng-click="kbnTopNav.toggle('filter')" aria-expanded="kbnTopNav.is('filter')" aria-label="time picker" data-test-subj="discoverNoResultsTimefilter"><span aria-hidden="true" class="kuiIcon fa-clock-o"></span> time picker</a> button in the top right corner of your screen.
+        I see you are looking at an index with a date field. It is possible your query does not match anything in the current time range, or that there is no data at all in the currently selected time range. Click the button below to open the time picker. For future reference you can open the time picker by clicking on the
+        <a
+          class="kuiButton kuiButton--primary kuiButton--small"
+          ng-click="kbnTopNav.toggle('filter')"
+          kbn-accessible-click
```

Comment:
Would you mind creating a new issue for this, with the "discuss" and "Accessibility" labels? This is an interesting idea and it seems like it would solve a good set of low-level issues! One tradeoff is that it's magic and relives the engineer of the responsibility of understanding and using ARIA well, which could be problematic when migrating Angular code to React.